package.json

```diff
@@ -4,16 +4,23 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --forceExit",
+    "test:watch": "jest --watchAll",
```

Comment:
Do we need the test:watch ??

Reply:
It was a dependency in the server-testing lesson today, so I added it